Make timeouts shorter which is more appropriate for client only use.
--- a/src/strongswan.conf
+++ b/src/strongswan.conf
@@ -1,6 +1,9 @@
 # strongswan.conf - strongSwan configuration file
 
 charon {
+	# shorter timeouts (defaults are 4.0/1.8)
+	retransmit_timeout = 3.0
+	retransmit_base = 1.4
 
 	# number of worker threads in charon
 	threads = 16
